_data/projects/Owasp-ZAP-fileupload-addon.yml

```diff
@@ -0,0 +1,14 @@
+---
+name: Owasp ZAP Fileupload Addon
+desc: OWASP ZAP add-on for finding vulnerabilities in File Upload functionality.
+site: https://github.com/SasanLabs/owasp-zap-fileupload-addon
+tags:
+- java
+- owasp-zap
+- dast
+- scan-rule
+- fileupload
+- scanner
+upforgrabs:
+  name: good first issue
+  link: https://github.com/SasanLabs/owasp-zap-fileupload-addon/labels/good%20first%20issue
```

Comment:
This is in another PR too. please close one or remove it from here. @preetkaran20